cirq-google/cirq_google/devices/known_devices.py

```diff
@@ -186,6 +181,35 @@ def create_device_proto_for_qubits(
     return out
 
 
+def populate_qubits_in_device_proto(
+    qubits: Collection[cirq.Qid], out: device_pb2.DeviceSpecification
+) -> None:
```

Comment:
Is it worth adding tests for the new functions in this file?

Reply:
I'm leaning towards no, because the behavior is tested by the callers, and also the logic here will eventually move inside GridDevice once SerializableDevice is deleted.